src/performance/type.d.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface ICalScore {
+  dpss: ICalScore[];
+  st: number;
+  els: Els;
+  root?: Element;
+}
+export type Els = Array<{
+  $node: Element;
+  st: number;
+  weight: number;
+}>;
+export type IPerfDetail = {
+  redirectTime: number | undefined; // Time of redirection
+  dnsTime: number | undefined; // DNS query time
+  ttfbTime: number | undefined; // Time to First Byte
+  tcpTime: number | undefined; // Tcp connection time
+  transTime: number | undefined; // Content transfer time
+  domAnalysisTime: number | undefined; // Dom parsing time
+  fptTime: number | undefined; // First Paint Time or Blank Screen Time
+  domReadyTime: number | undefined; // Dom ready time
+  loadPage: number | undefined; // Page full load time
```

Comment:
@Fine0830 In [BrowserPerf.proto](https://github.com/apache/skywalking-data-collect-protocol/blob/b2f381e63702bc43216ef5576637195102302c6e/browser/BrowserPerf.proto#L66), I changed `loadPage` to `loadPageTime`,